src/winit_input_helper.rs

```diff
@@ -255,12 +255,13 @@ impl WinitInputHelper {
         }
     }
 
-    /// Returns `0.0` if the mouse is outside of the window.
+    /// Returns `(0.0, 0.0)` if the mouse is outside of the window.
     /// Otherwise returns the amount scrolled by the mouse during the last step.
-    pub fn scroll_diff(&self) -> f32 {
+    /// Returns (vertically, horizontally)
+    pub fn scroll_diff(&self) -> (f32, f32) {
         match &self.current {
-            Some(current) => current.scroll_diff,
-            None => 0.0,
+            Some(current) => (current.y_scroll_diff, current.x_scroll_diff),
```

Comment:
Usually a vector goes: (x, y)
ordering it (y, x) would be very unexpected for the user

Reply:
Swapped order